source/developers/transforms-with-object-lambda.rst

```diff
@@ -121,6 +121,8 @@ MinIO also supports the following environment variables for authenticated webhoo
 
 Restart MinIO to apply the changes.
 
+You can also configure Object Lambda with the MinIO Client :mc-cmd:`mc` command line tool.
+For more information, see :ref:`minio-server-envvar-object-lambda-webhook`.
```

Comment:
I don't want to rework this page, so just dropping a reference here for interested readers.